requirements.txt

```diff
@@ -0,0 +1,6 @@
+geoip2>=2.8.0
```

Comment:
It is sort of unfortunate that we have to do this for Synk. I think this sort of thing is generally discouraged nowadays for libraries.